### ISSUE TITLE
Python default build-flow "python-pip" fails in Windows

### ISSUE THREAD

**Issue by RonWang** [alexa/ask-cli#123]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

## Current Behavior
```
> ask deploy
==================== Deploy Skill Metadata ====================
Skill package deployed successfully.
Skill ID: amzn1.ask.skill.b1c73924-407e-4752-9e32-f16470af61c0

New-Py3Venv : The term 'Write-Log' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is
At D:\Github\ask-cli\lib\builtins\build-flows\python-pip\build.ps1:147 char:1
+ ~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (Write-Log:String) [New-Py3Venv], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException,New-Py3Venv

[Error]: [Error]: Build Scripts failed with non-zero code: 1.
Write_Log : The term 'Write_Log' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is
correct and try again.
At line:1 char:1
+ Write_Log
+ ~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (Write_Log:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

And after changing Write_Log to Show_Log this happens:
```
> ask deploy
==================== Deploy Skill Metadata ====================
[Warn]: The hash of current skill package folder does not change compared to the last deploy hash result, CLI will skip the deploy of skill package.
Skill ID: amzn1.ask.skill.b1c73924-407e-4752-9e32-f16470af61c0

==================== Build Skill Code ====================
Invoke-Expression : The term 'python3' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path
is correct and try again.
At D:\Github\ask-cli\lib\builtins\build-flows\python-pip\build.ps1:86 char:9
+         Invoke-Expression -Command "python3 -m venv venv"
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (python3:String) [Invoke-Expression], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException,Microsoft.PowerShell.Commands.InvokeExpressionCommand

[Error]: [Error]: Build Scripts failed with non-zero code: 1.
```

## Steps to Reproduce (for bugs)

ask deploy a skill with python and windows.

## Possible Solution
Correct the default powershell script for python-pip build flow.

## Your Environment and Context

* ask-cli version: 2.0.1
* Operating System and version: Win10

**Comment by RonWang** [comment]
PR raised to fix this issue: https://github.com/alexa/ask-cli/pull/124

**Comment by RonWang** [comment]
This is now solved by release v2.1.0 https://github.com/alexa/ask-cli/releases/tag/v2.1.0
